qiita_db/metadata_template/base_metadata_template.py

```diff
@@ -1185,6 +1193,8 @@ def update(self, md_template):
                     "There are no differences between the data stored in the "
                     "DB and the new data provided",
                     QiitaDBWarning)
+                return
```

Comment:
Yes, that is the test that makes sure that the warning is not raised, but the test for this specific warning is in `test_update_equal`